lib/rage/cli.rb

```diff
@@ -25,6 +25,40 @@ def server
 
       ::Iodine.start
     end
+
+    desc 'routes', 'List all routes.'
+    def routes
+      # the result would be something like this:
+      # +----------------------+----------------------+----------------------+
+      # | Action               | Verb                 | Path                 | Controller#Action
+      # | index                | GET                  | /                    | application#index
+
+      # load config/application.rb
+      require File.expand_path('config/application.rb', Dir.pwd)
+      # load config/routes.rb
+      require File.expand_path('config/routes.rb', Dir.pwd)
```

Comment:
you only need to load the `config/application` file, as it [loads all routes automatically](https://github.com/rage-rb/rage/blob/master/lib/rage/setup.rb). Also, let's move this require call into a method, as this is the only task here that needs access to the application code.